applications/fishing-map/src/data/datasets.ts

```diff
@@ -1,5 +1,5 @@
 // TODO: rename this to use 4WINGS DATASET TYPE
-export const FISHING_DATASET_TYPE = '4wings:v1'
+export const FOURWINGS_DATASET_TYPE = '4wings:v1'
```

Comment:
Apparently this constant is also defined in our react-hooks https://github.com/GlobalFishingWatch/frontend/blob/fishingmap/datasets-upload/packages/react-hooks/src/use-dataviews-layers/use-dataviews-layers.ts#L12. Should it be renamed too to keep consistency?

Reply:
Good point! I moved the DatasetType from type to enum to centralise the value and use it everywhere, see commit: https://github.com/GlobalFishingWatch/frontend/commit/e6dfab02f7bfb6a051fba402835a135b57a10f1c Note: I didn't change Amalthea as it is deprecated and will be removed soon